ccd-definition/CaseEvent.json

```diff
@@ -425,5 +425,19 @@
     "ShowSummary": "N",
     "ShowEventNotes": "Y",
     "EndButtonLabel": "Save and continue"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "hearingBookingDetails",
+    "Name": "Add hearing details",
+    "Description": "Add hearing booking details to a case",
+    "DisplayOrder": 1,
```

Comment:
where does displayOrder 1 have this sit in the dropdown?

Reply:
First in the list where should it be ?

Reply:
cool 👍